.github/workflows/jacktrip.yml

```diff
@@ -61,13 +61,13 @@ jobs:
 
           - name: Linux-x64-qmake-gcc-static-nogui
             release-name: Linux-x64-nogui
-            runs-on: ubuntu-18.04
+            runs-on: ubuntu-20.04
             system-rtaudio: false
             bundled-rtaudio: false
             nogui: true
             weakjack: false
-            static-qt-version: 5.12.11
-            qt-static-cache-key: 'v22' # we use the same key as the main Linux build
+            static-qt-version: 5.15.2
```

Comment:
The reason for using Qt 5.12 initially was to make the build compatible with Ubuntu 20.04 (the latest LTS version at the time of this addition). Static qt 5.15 did not run on this version when I last tested it. If possible, I think it would be preferable to keep building with 5.12 to support older OSs...

Reply:
I thiiiiink this change was a mistake. A `nogui` build shouldn't need any Qt updating. Going to double check.

Reply:
I will say there are Qml features being used (required properties, as an example) that require 5.15 for VS builds. 

I know you're asking specifically about Linux `nogui` builds, but here's some added context.

5.15 still supports Windows 7, but it drops support for macOS Sierra, which when I looked into it was estimated to be used by about 2% of the existing active macOS install base. That data didn't include Monterey which means it is probably less than that.

Reply:
Would be great to switch to 5.15. #320 was the PR that downgraded to 5.12. @dyfer Do you remember what the compatibility problem was? Maybe we can fix it?

Reply:
[This](https://github.com/jacktrip/jacktrip/pull/320#issuecomment-857121580) looks like the the static xcb plugin was missing.

Reply:
@dyfer Do you remember if you tried [adding qxcb or similar to QTPLUGIN_platforms](https://doc.qt.io/qt-5/plugins-howto.html#static-plugins)?

Reply:
Looks like installing `libxcb-util-dev` fixed it for some people. [Seems like a hidden requirement](https://forum.qt.io/topic/125582/qt-5-15-2-do-not-build-libqxcb-so-but-at-the-same-time-qt-5-14-0-successfully-build-libqxcb-so). This is 18.04, but worth a shot.

Reply:
This is _extremely_ fresh 20.04 (booted off usb) attempting to run the static shared build with rtaudio. I had to install `libxcb-xinerama0-dev` and it still loads to a blank screen for 5.12/5.15 incompatibility reasons (see last screenshot).

![Screenshot from 2022-05-05 22-29-53](https://user-images.githubusercontent.com/5567285/167035667-da06972b-c23e-44bf-9940-19d76c139b85.png)
![Screenshot from 2022-05-05 22-29-21](https://user-images.githubusercontent.com/5567285/167035672-2d63efd7-76ec-467e-96ac-d882cd706023.png)
![Screenshot from 2022-05-05 22-35-03](https://user-images.githubusercontent.com/5567285/167036070-10006a6f-c944-4f2b-bb3a-23e85ee04dca.png)